tools/api_check/lib/apicheck.dart

```diff
@@ -53,14 +51,14 @@ List<String> getDartClassFields({
   return fields;
 }
 
-NodeList<ClassMember>? _getMembers(String className, CompilationUnitMember unitMember) {
+NodeList<ClassMember> _getMembers(String className, CompilationUnitMember unitMember) {
```

Comment:
This likely will have to be typed `List<ClassMember>`. Same for the variable in line 38. I think after that, we got this.

Reply:
This is the hardest live programming I've done. Got it.

Reply:
:) Thanks for sticking with it. I think we got it now. Let's see what the checks say!

Reply:
It passed. Finally!